nodeup/pkg/model/tests/golden/multiple-sa-issuers/cluster.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: kops.k8s.io/v1alpha2
+kind: Cluster
+metadata:
+  name: multisaissuer.example.com
+spec:
+  kubernetesApiAccess:
+  - 0.0.0.0/0
+  channel: stable
+  cloudProvider: aws
+  configBase: memfs://clusters.example.com/multisaissuer.example.com
+  etcdClusters:
+  - cpuRequest: 200m
+    etcdMembers:
+    - instanceGroup: master-us-test-1a
+      name: us-test-1a
+    memoryRequest: 100Mi
+    name: main
+    provider: Manager
+    backups:
+      backupStore: memfs://clusters.example.com/multisaissuer.example.com/backups/etcd-main
+  - cpuRequest: 100m
+    etcdMembers:
+    - instanceGroup: master-us-test-1a
+      name: us-test-1a
+    memoryRequest: 100Mi
+    name: events
+    provider: Manager
+    backups:
+      backupStore: memfs://clusters.example.com/multisaissuer.example.com/backups/etcd-events
+  iam: {}
+  kubelet:
+    anonymousAuth: false
+  kubernetesVersion: v1.28.0
+  masterPublicName: api.multisaissuer.example.com
+  networkCIDR: 172.20.0.0/16
+  kubeAPIServer:
+    serviceAccountIssuer: https://huh.com/multisaissuer.example.com
+    additionalServiceAccountIssuers:
+    - https://foobar.com
+    - https://aaa.com
+    - https://hello.com
+    - https://dorld.com
```

Comment:
Could this be put in the complex test or one of the IRSA ones? I don't think we gain much by adding another test.